src/main/kotlin/com/sourcegraph/cody/auth/SelectOneOfTheAccountsAsActive.kt

```diff
@@ -1,17 +1,29 @@
 package com.sourcegraph.cody.auth
 
+import com.intellij.openapi.diagnostic.Logger
 import com.intellij.openapi.project.Project
 import com.sourcegraph.cody.CodyToolWindowContent
+import com.sourcegraph.cody.config.CodyAccount
 import com.sourcegraph.cody.config.CodyAuthenticationManager
 import com.sourcegraph.cody.config.getFirstAccountOrNull
 import com.sourcegraph.cody.initialization.Activity
 
 class SelectOneOfTheAccountsAsActive : Activity {
+
+  private val logger = Logger.getInstance(SelectOneOfTheAccountsAsActive::class.java)
+
   override fun runActivity(project: Project) {
-    val codyAuthenticationManager = CodyAuthenticationManager.instance
-    if (codyAuthenticationManager.getActiveAccount(project) == null) {
-      val newActiveAccount = codyAuthenticationManager.getAccounts().getFirstAccountOrNull()
-      codyAuthenticationManager.setActiveAccount(project, newActiveAccount)
+    val authManager = CodyAuthenticationManager.instance
+    val token = System.getenv("SOURCEGRAPH_TEST_TOKEN")
```

Comment:
I think it is worth to rename it to `testToken` - to strongly reflect it's purpose. Or maybe even extract the test account set up to a separate private method.

Reply:
You're right. I've cleaned this code now.